Tuist.swift

```diff
@@ -4,6 +4,7 @@ let config = Config(
     fullHandle: "tuist/tuist",
     swiftVersion: .init("5.10"),
     generationOptions: .options(
-        optionalAuthentication: true
+        optionalAuthentication: true,
+        disableSandbox: true
```

Comment:
our `tuist/tuist` repo should go against our strongly recommended guidelines – and the sandbox is one of them. Let's figure out a way, so we don't have to disable it.

Reply:
I changed the approach and forgot to revert this one. It should be fine now.